cellprofiler/modules/mergelabels.py

```diff
@@ -0,0 +1,106 @@
+# coding=utf-8
+
+"""
+MergeLabels
+===========
+
+**MergeLabels** merges labels below a certain threshold into its most prevalent, adjacent neighbor.
```

Comment:
I think you should add a note here stating that objects below the threshold with no adjacent neighbors will be removed.